documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+---
+title: Serenity/JS 2.x Migration Guide
+layout: handbook.hbs
+---
+
+# Serenity/JS 2.x Migration Guide
+
+## Introduction
+
+This guide is designed for people that have built Serenity/JS 1.x projects and wish to upgrade to Serenity/JS 2.x. The upgrade path is relatively straightforward as the main classes (`Actor`, `Task`, `Interaction` and `Question`) 
+are backwardly compatible. 
+
+There are some [example mini-projects](https://github.com/jan-molak/serenity-js/tree/2.x/examples) already created in version 2.x format that demonstrate the major features for each supported framework, 
```

Comment:
Could we update the link to https://github.com/serenity-js/serenity-js/tree/master/examples and mention the template repositories at https://github.com/serenity-js ?

Reply:
👍 done